packages/arb-ts/src/lib/l2Bridge.ts

```diff
@@ -214,6 +219,20 @@ export class L2Bridge {
     return this.arbTokenBridge.calculateBridgedERC20Address(erc20L1Address)
   }
 
+  public getERC20L1Address(erc20L2Address: string) {
+    try {
+      const arbERC20 = StandardArbERC20Factory.connect(
+        erc20L2Address,
+        this.l2Signer
+      )
+      return arbERC20.l1Address()
+    } catch (e) {
+      console.warn('Could not get L1 Address')
+
+      return
```

Comment:
Do we want to throw an error or return undefined?

Reply:
having a check that doesn't throw proved useful; the consumer of the API can do w/ this what they want (throw, chill, etc.)